flybywiresim/aircraft
# fix: brake temperature sometime starts at zero

Fixes #8153

## Summary of Changes
Rewritten the brake temperature system and fix the original bug by initializing to ambiant temperature. Also, light up all "hottest brake" indication when they are more than one "hottest brake" (cf. [discussion with Tolip](https://discord.com/channels/738864299392630914/738864300818432023/1136921224665116682))

- [x] Need to add some documentation

## Testing instructions

- "Zero temperature" fix:

1.   try starting on runway or cold&dark on multiple airports and meteo conditions and check brakes temperature on SD WHEEL page: it should never be 0 if ambiant temperature exceeds 5 degrees.

- "All hottest brake light up":

1. Start on runway, throttle up then brake hard until multiple brakes exceeds 100°. If they are many with the same temperature, they should all have the green arc over the number

- Non regression tests:

1. After heating up the brakes as above, let the aircraft without input and it should slowly cool down the brakes
2. After heating up the brakes as above, let the aircraft without input, push the Brake Fan button and it should cool down the brakes faster 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
